### ISSUE TITLE
[Bug] GPU transforms are not fed GPU data for keys other than 'data'

### ISSUE THREAD

**Issue by patmjen** [PhoenixDL/rising#113]
**Description**
If I supply GPU transforms that operate on multiple keys to a `DataLoader`, then only data for the `data` key is transferred to the GPU prior to feeding it to the transforms. For example, if I'm doing spatial transforms (such as flipping), I want to flip both the `data` and `labels` - and I want it to happen on the GPU for speed.

The error seems to happen at line 187 of `loading/loader.py`:
https://github.com/PhoenixDL/rising/blob/7da711c27ae23eda3b27b2168aac9505c30e928f/rising/loading/loader.py#L183-L190
No `keys` argument is given to `ToDevice` so it uses its default which is `keys = ('data',)`, c.f. [`transforms/tensor.py:52`](https://github.com/PhoenixDL/rising/blob/7da711c27ae23eda3b27b2168aac9505c30e928f/rising/transforms/tensor.py#L47-L65).

**Environment**
* OS: Windows 10
* Python version: Python 3.8.5
* `rising` version: 0.2.0post0

**Reproduction**
```python
from rising.loading import DataLoader
from rising.transforms.abstract import BaseTransform


def check_on_gpu(x):
    assert(x.is_cuda)
    return x


class GpuChecker(BaseTransform):
    def __init__(self, keys=('data',)):
        super().__init__(augment_fn=check_on_gpu, keys=keys)


if __name__ == '__main__':
    # Data is definitely on CPU...
    data = [
        { 'data': 1, 'label': 1 },
        { 'data': 2, 'label': 2 },
        { 'data': 3, 'label': 3 }
    ]

    # This will work
    print('Only data')
    loader = DataLoader(data, gpu_transforms=GpuChecker())
    for x in loader:
        print(x)

    # This will crash
    print('Both data and labels')
    loader = DataLoader(data, gpu_transforms=GpuChecker(('data', 'label')))
    for x in loader:
        print(x)


```

**Comment by justusschock** [comment]
Hey @patmjen ,

thanks for the excellent reproduction example and sorry for the very long turnaround. This should be fixed in https://github.com/PhoenixDL/rising/commit/2b951cf62b090c733e0b9535042cb0ef6443d3cd via the new argument for the loader and will be released very soon
